src/lib/ast/empty.ts

```diff
@@ -0,0 +1,31 @@
+import { Skill } from "../types";
+import { SkillExpression } from "./skillExpression";
+import { Variable } from "./variable";
+import { SkillsRelations } from "./skillsRelation";
+
+// And skill expression
+export class Empty extends SkillExpression {

```

Comment:
- For what is the empty expression needed?
- Comment was copied from `AND` class.

Reply:
An empty skill expression used in case the learning skill does not have any requirement.
The evaluation of this skill expression is always true.
We use this to ensure consistency across all learning units.

Reply:
Wouldn't it be easier to make a preconditions in general **optional**? This way we need to check that a condition is present, but we do not need to create empty objects that need to be evaluated.